packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -239,6 +233,137 @@ class OpenSettingsAction extends StatelessWidget {
   }
 }
 
+class DevToolsAboutDialog extends StatefulWidget {
+  @override
+  _DevToolsAboutDialogState createState() => _DevToolsAboutDialogState();
+}
+
+class _DevToolsAboutDialogState extends State<DevToolsAboutDialog> {
+  FlutterVersion _flutterVersion;
+  InfoController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller = InfoController(
+      onFlutterVersionChanged: (flutterVersion) {
+        if (!mounted) return;
+        setState(
+          () {
+            _flutterVersion = flutterVersion;
+          },
+        );
+      },
+    )..entering();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+
+    return AlertDialog(
+      actions: <Widget>[
+        FlatButton(
+          onPressed: () {
+            Navigator.of(context, rootNavigator: true).pop('dialog');
+          },
+          child: const Text('CLOSE'),
+        ),
+      ],
+      content: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text('DevTools', style: textTheme.headline6),
+          const PaddedDivider(
+            padding: EdgeInsets.only(bottom: denseRowSpacing),
+          ),
+          const SelectableText('DevTools version ${devtools.version}'),
+          //
+          const SizedBox(height: defaultSpacing),
+          Text('Connected Device Info', style: textTheme.headline6),
+          const PaddedDivider(
+              padding: EdgeInsets.only(bottom: denseRowSpacing)),
+          _createDeviceInfo(context, textTheme, _flutterVersion),
+          //
+          const SizedBox(height: defaultSpacing),
+          Text('Feedback', style: textTheme.headline6),
+          const PaddedDivider(
+            padding: EdgeInsets.only(bottom: denseRowSpacing),
+          ),
+          Wrap(
+            children: [
+              const Text('Encountered an issue? Let us know at '),
+              _createFeedbackLink(context, textTheme),
+              const Text('.')
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _createDeviceInfo(
+    BuildContext context,
+    TextTheme textTheme,
+    FlutterVersion flutterVersion,
+  ) {
+    if (flutterVersion == null) {
+      return const Text('No Flutter device connected.');
+    }
+
+    final versions = {
+      'Flutter': flutterVersion.version,
+      'Framework': flutterVersion.frameworkRevision,
+      'Engine': flutterVersion.engineRevision,
+      'Dart': flutterVersion.dartSdkVersion,
+    };
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        for (var name in versions.keys)
+          Padding(
+            padding: const EdgeInsets.only(bottom: 8.0),
```

Comment:
nit: the output looks great but having padding only at the bottom here is a bit surprising.
Can you instead add padding of 4.0 to both the top and bottom and tweak the padding elsewhere so the UI still looks the same as it does now?